src/shared/services/UnreadCounterService.ts

```diff
@@ -79,10 +83,24 @@ export class UnreadCounterService {
     }
   }
 
+  public async updatePendingFollows() {
+    if (this.shouldUpdate) {
+      // TODO: remove community_id param
```

Comment:
I see this is part of the back-end PR you have. Anyway you can probably remove the comment because a newly generated lemmy-js-client will annoy us about this.